### PR TITLE
Feature/function on save

### PR DESCRIPTION
**Pull request by ASands17**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
This feature adds an event listener to ensure duplicate posters are not saved on the display posters page. It allows posters to be saved and displayed on the saved posters page. 
  <li>What does it fix?</li>
n/a
  <li>Is this a bug or a feature?</li>
Feature
  <li> Does it break any existing functionality or force me to update to a new version?</li>
No
  <li>How has it been tested?</li>
We have tested this using devTools and the webpage. 
</ul>

<h2>Type of Change</h2>

_Delete options that are not relevant_

- [X] **New Feature**

<h2>How has this been tested?</h2>

- [X] **Dev Tools**
- [X] **Website**

<h2>Checklist</h2>

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new bugs
- [X] I have tagged project partners for review & it has been reviewed

### CODE REVIEW

**Review by Neil-B-Linton — COMMENTED** [review]
This is some amazing code!! I love this, you did a really good job!

File: src/main.js [hunk 1]
```diff
@@ -212,6 +216,31 @@ function showNewView() {
...
+    display += `<article class="mini-poster">
+      <img class="poster-img" id=${savedPosters[i].id} src="${savedPosters[i].imageURL}" alt="nothin' to see here">
+      <h2 class="poster-title">${savedPosters[i].title}</h2>
+      <h4 class="poster-quote">${savedPosters[i].quote}</h4>
+    </article>`
+  }
+  savedPosterGrid.innerHTML = display;
+};
+
+function avoidDuplicates() {
```
Comment (JSweet314) [inline src/main.js:233]: Could this function be named differently to better describe what it does? Appears to save unique items but not clear from where it is called (as event handler function for save button click event)

File: src/main.js [hunk 2]
```diff
@@ -212,6 +216,31 @@ function showNewView() {
   saveView.classList.add("hidden")
 };
 
+//Save poster function
+
+function savePoster() {
```
Comment (JSweet314) [inline src/main.js:221]: This function seems to have less to do with saving posters, and more to do with displaying them.

**Review by JSweet314 — COMMENTED** [review]
Nice work! 

When it comes to naming functions, think about what the function does as well as where it may be called. Good function names make it easier for you and other developers to know what is going on when a function is invoked without necessarily having to look at how it is implemented.

File: src/main.js (hunk 1 above)
Comment (ASands17) [reply src/main.js:233]: I see what you mean Jon. I've updated this function name to be more accurate.

File: src/main.js (hunk 2 above)
Comment (ASands17) [reply src/main.js:221]: Great point. I have updated this function name to be displayPoster since that's more clear.

**Comment by ASands17** [comment]
Thank you for the feedback Jon!

**Pull request merged**